rpc/client/client.go

```diff
@@ -769,6 +770,28 @@ func (c *Client) Status(ctx context.Context) (*ctypes.ResultStatus, error) {
 	return result, nil
 }
 
+// SyncInfo returns detailed information about current sync status.
+func (c *Client) SyncInfo() *ResultSyncInfo {
```

Comment:
thinking, as to my previous comments if all this info just better off written to a gauge